plenum/test/batching_3pc/catch-up/test_catchup_during_3pc.py

```diff
@@ -0,0 +1,30 @@
+import pytest
```

Comment:
A general comment on the next 2 tests, you can simulate the conditions of these 2 tests by delaying COMMITs, and delaying COMMITs for sometime and then resetting delays (simulating a bad network and simulating repair of the bad network) respectively, that removes the reliance on patching the methods. Also patched methods hinder refactoring.